charts/sda-svc/templates/_helpers.yaml

```diff
@@ -101,25 +101,29 @@ Create chart name and version as used by the chart label.
   {{- .Values.global.secretsPath -}}
 {{- end -}}
 
+{{- define "configPath" -}}
+{{- ternary .Values.global.secretsPath (printf "%s/%s" (.Values.global.secretsPath | trimSuffix "/") (.Values.global.confFilePath | trimPrefix "/")) (empty .Values.global.confFilePath) -}}
```

Comment:
Concatenation path using `printf "%s/%s"` requires `.Values.global.secretsPath` to be non empty. It will be nice to add a check that `.Values.global.secretsPath` should not be empty.